constellations/orion/src/repos/account_repo_test.go

```diff
@@ -176,14 +176,15 @@ func Test_SelectAllNegativeBalances(t *testing.T) {
 }
 
 //
-// Create
+// Create User and Account
 //
-func TestInsertAccount(t *testing.T) {
+func Test_InsertAccountAndUser(t *testing.T) {
```

Comment:
I removed all underscores from test names, so this can just be `TestInsertAccountAndUser`